engine/src/main/java/org/terasology/rendering/dag/nodes/FinalPostProcessingNode.java

```diff
@@ -104,9 +105,9 @@ public void initialise() {
     public void process() {
         PerformanceMonitor.startActivity("rendering/finalPostProcessing");
 
-        ocUndistorted = frameBuffersManager.getFBO("ocUndistorted");
-        sceneFinal = frameBuffersManager.getFBO("sceneFinal");
-        sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
+        ocUndistorted = dynamicFBM.getFBO(OC_UNDISTORTED_URN);
+        sceneFinal = dynamicFBM.getFBO(DefaultDynamicFBOs.Final.getResourceUrn());
+        sceneOpaque = dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
```

Comment:
Why here and in other nodes you use DynamicFBM.getFBO to obtain default FBOs that can be easily referenced through say DefaultDynamicFBOs.Final? Of course it is right to require them on initialization and update them when notified of changes, if necessary.